samples/MvcSample.Web/bower.json

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "MvcSample.Web",
+    "description": "Web site demonstrating various MVC features.",
+    "private": true,
+    "dependencies": {
+        "jquery": "1.11.2"
```

Comment:
nit: since this is required by FormUrlEncoded/Index.cshtml and not MVC.js, suggest adding a comment to MVC.js indicating it has no jQuery dependencies.